From b4d601658209237d932ca3a6574eeb743be1ede9 Mon Sep 17 00:00:00 2001
From: Felix Yan <felixonmars@archlinux.org>
Date: Fri, 3 Nov 2017 16:35:04 +0800
Subject: [PATCH] Add some path workarounds to Arch path configuration

This avoids errors about missing variables in jail configuration. See downstream report at https://bugs.archlinux.org/task/48540
---
 config/paths-arch.conf | 9 +++++++++
 1 file changed, 9 insertions(+)

--- ./config/paths-arch.conf
+++ ./config/paths-arch.conf
@@ -20,6 +20,15 @@ mysql_log = /var/log/mariadb/mariadb.log
 
 roundcube_errors_log = /var/log/roundcubemail/errors
 
+# Workaround missing variables in jail paths
+syslog_authpriv =
+syslog_mail =
+syslog_mail_warn =
+syslog_user =
+syslog_ftp =
+syslog_daemon =
+syslog_local0 =
+
 # These services will log to the journal via syslog, so use the journal by
 # default.
 syslog_backend = systemd
